samcli/commands/init/interactive_init_flow.py

```diff
@@ -163,9 +168,12 @@ def _generate_from_use_case(
     location = templates.location_from_app_template(package_type, runtime, base_image, dependency_manager, app_template)
 
     final_architecture = get_architectures(architecture)
+    lambda_supported_runtime = (
```

Comment:
This `lambda_supported_runtime` is not used below, where we call `is_dynamic_schemas_template` or in the `generate_summary_message`. Is that expected?